event_listener_test.go

```diff
@@ -133,6 +134,9 @@ func (f loggingFile) Sync() error {
 
 // Verify event listener actions, as well as expected filesystem operations.
 func TestEventListener(t *testing.T) {
+	if unsafe.Sizeof("") != 16 {
```

Comment:
I think this check for 64bit is weird. I mean, it works, but why look at `""` specifically?

Reply:
I agree it is weird. The reason is that the test failure stems from exactly this reason, but more specifically this could be used instead, which might be clearer: 
```suggestion
	if unsafe.Sizeof(sstable.Reader{}) != 704 {
```

Reply:
Due to this line: https://github.com/cockroachdb/pebble/blob/master/table_cache.go#L165

Reply:
... and the expectation here: https://github.com/cockroachdb/pebble/blob/master/testdata/event_listener#L207